src/components/utils/AdvertisingConfigPropType.js

```diff
@@ -85,6 +85,9 @@ export default PropTypes.shape({
         )
     ),
     slots: PropTypes.arrayOf(AdvertisingSlotConfigPropType),
+    outOfPage: PropTypes.shape({
```

Comment:
⚠️ **this is incorrect** – there is no config property `outOfPage`, there is `outOfPageSlots`, which is an array of shapes with a single property `id`

Reply:
missed it, and changed it